docs/src/user_docs/stdlib/collections.md

```diff
@@ -0,0 +1,22 @@
+# Collections
+Namespace `std::crypto` contains modules for commonly authenticated data structures.
+
+## Merkle Mountain Range
+Module `std::collections::mmr` contains procedures for manipulating [Merkle Mountain Ranges](https://github.com/opentimestamps/opentimestamps-server/blob/master/doc/merkle-mountain-range.md) data structure which can be used as an append-only log.
```

Comment:
nit: I would say "Merkle Mountain Range data structures, which can be..." or "Merkle Mountain Ranges, which can be..."